templates/controltower.yaml

```
@@ -0,0 +1,251 @@
+AWSTemplateFormatVersion: 2010-09-09
+Metadata:
+  'AWS::CloudFormation::Interface':
+    ParameterGroups:
+      - Label:
+          default: Required parameters
+        Parameters:
+          - ObserveCustomer
+          - ObserveToken
+          - ManagedResourcePrefix
+      - Label:
+          default: Observe Lambda Configuration
+        Parameters:
+          - LambdaVersion
+          - LambdaReservedConcurrentExecutions
+          - LambdaTimeout
+          - LambdaMemorySize
+          - LambdaS3CustomRules
+          - LambdaS3ReadAny
+Parameters:
+  ObserveCustomer:
+    Type: String
+    Description: Observe Customer ID
+    AllowedPattern: '\d+'
+    ConstraintDescription: 'Customer ID can only contain digits'
+  ObserveToken:
+    Type: String
+    NoEcho: true
+    Description: Observe Ingest Token
+    AllowedPattern: '\S+'
+    ConstraintDescription: 'Token cannot contain spaces'
+  ObserveDomain:
+    Type: String
+    Default: observeinc.com
+    Description: Observe domain to submit data to
+  ManagedResourcePrefix:
+    Type: String
+    Description: The ControlTower ManagedResourcePrefix
+    Default: aws-controltower
+  LambdaVersion:
+    Type: String
+    Default: latest
+    Description: Observe lambda function version
+  LambdaReservedConcurrentExecutions:
+    Type: Number
+    Default: 100
+    Description: The number of simultaneous executions to reserve for the function. Set to -1 to not reserve concurrent executions.
+  LambdaTimeout:
+    Type: Number
+    Default: 60
+    Description: >-
+      The amount of time that Lambda allows a function to run before stopping
+      it. The maximum allowed value is 900 seconds.
+  LambdaMemorySize:
+    Type: Number
+    Default: 128
+    MinValue: 128
+    MaxValue: 10240
+    Description: >-
+      The amount of memory that your function has access to. The value must be a
+      multiple of 64 MB.
+  LambdaS3CustomRules:
+    Type: String
+    Default: ""
+    Description: >-
+      A base64-encoded JSON array of rules which override how S3 objects are
+      submitted to Observe.
+  LambdaS3ReadAny:
+    Type: String
+    Default: false
+    Description: >-
+      Grant the lambda S3 read only access to all buckets. This simplifies the
+      process of subscribing new buckets to the lambda. The lambda function
+      will only ever attempt to read objects for which it has received
+      notifications. By default, we restrict the lambda to only being allowed
+      to access the S3 bucket managed by the cloudformation template.
+    AllowedValues:
+      - true
+      - false
```

Double check AWS docs but I think these shoudl be strings
```suggestion
    AllowedValues:
      - "true"
      - "false"
```